lib/testscript_engine/cli.rb

```diff
@@ -56,7 +56,7 @@ def self.start
       @testreport_path = options["testreport_path"] if options["testreport_path"]
       @load_non_fhir_fixtures = options["nonfhir_fixture"] if options["nonfhir_fixture"]
       
-      runnable = nil
+      runnable = options["runnable"] unless options["runnable"]
```

Comment:
I think we will want to be able to specify a list of runnables to execute, either at the command line or in the YAML config file. You can do this now by organizing folders, but say you want to pick out a few tests from a larger suite that are relevant to a particular server.  Wondering about 
1. making the runnable setting in the config file a list
2. rather than having an explicit flag for runnable in the command line treating everything without a flag as a runnable specification

Thoughts?

Reply:
It wouldn't be hard to program to set a list of testscript files as runnables in both config.xml or CLI arguments. I wonder how practical it will be used by arguments. If I want to test more than three specific testscript files, I probably wouldn't do it in CLI since it's overwhelming to enter. But it's all by user preference.

Can you elaborate #2?

Reply:
1. I'm ok with for now saying if you want to put in a list of runnables, you have to use the config file and can only specify one (or all) at the command line. Just make sure that's clear in the documentation if you go that direction. See the next list entry for an alternative.
2. in CLIs some arguments have flags, but others are just the expected inputs. For example `mkdir [DIR_NAME]`.  If you look at the man page for mkdir (`man mkdir`), you'll see that it describes the command as `mkdir [-pv] [-m mode] directory_name ...`, which means
  - `mkdir` is the command
  - you have some optional flagged inputs
  - each of any further inputs are directory names to create.

The last bullet is where I would suggest having runnables go.  You'd describe our command as `bin/testscript_engine [options ...] runnable_name ...`